examples/integrations/gcp/src/coordinator/pipelines/images/bounding_box.py

```diff
@@ -5,48 +5,42 @@
 from google.cloud import aiplatform
 from google.cloud.aiplatform.models import Model
 
-from pipelines.types import Pipeline, CustomJob, JobStatus, JobState, Job
+from pipelines.types import Pipeline, JobStatus, JobState, Job
 
 logger = logging.getLogger("uvicorn")
 
 
-class BoundingBoxETL(CustomJob):
+class BoundingBoxETL(Job):
+    container_name = "gcr.io/sandbox-5500/training-repo/bounding_box_etl"
```

Comment:
what's the plan longer term for these container URIs? will they be passed as arguments? right now a user would have to edit these both in the docker-compose file and in all of the etl classes, right?

Reply:
oops. yeah the project name should come from an env var. I just updated it. Thanks!